frontend/src/components/ReportSelector/reportSelectorConfig.tsx

```diff
@@ -396,7 +396,7 @@ export const categoriesAndTypes: IReportCategoriesAndTypes = [
       {
         value: "Tab_6_rpt_CA_ConsultingServiceUtilization-FY-Summary-byPortfolio",
         label: "Contractor Utilization summary by Portfolio",
-        description: `Report based on invoiced amount towards each type of resource (Application Design-Development, Application Management, Architecture, Business Analysis and Process Consultation etc. Based on the Competencies in the SRI).`,
+        description: `Retrieves Summary stats report for types of contract resources we have engaged on CO's. Information based on data from 14-15 forward. Shows distribution over the various Portfolios.`,
```

Comment:
good catch - the Purpose and Description on the `ReportPriorityBuild` spreadsheet from the LAN seem to vary wildly, and this is a better fit than the actual description listed for the tab 6 report